src/components/booksList.js

```diff
@@ -11,14 +11,20 @@ const mapDispatchToProps = dispatch => ({
   removeBook: book => dispatch(RemoveBook(book)),
 });
 
-const booksList = ({ books }) => (
-  <table>
-    <tbody>
-      {
-        books.map(book => (<Book key={book.id} book={book} />))
-      }
-    </tbody>
-  </table>
-);
+const booksList = ({ books, removeBook }) => {
+  const removeClick = book => {
+    removeBook(book);
+  };
+
+  return (
+    <table>
+      <tbody>
+        {
+          books.map(book => (<Book key={book.id} book={book} remove={removeClick} />))
```

Comment:
* [ ] it passes `handleRemoveBook` to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
let use `handleRemoveBook`, as required. This makes it more semantic and easy to understand than just remove.